blazor/ClientSideExample/ClientSideExample/ViewModels/GreetingViewModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Linq;
+using System.Reactive;
+using System.Reactive.Linq;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+
+namespace ClientSideExample.ViewModels
+{
+    public class GreetingViewModel : ReactiveObject
+    {
+        public ReactiveCommand<Unit, Unit> Clear { get; }
+
+        [ObservableAsProperty]
+        public bool CanClear { get; }
+
+        [ObservableAsProperty]
+        public string Greeting { get; }
+
+        [Reactive]
+        public string Name { get; set; } = string.Empty;
+
+        public GreetingViewModel()
+        {
+            var canClear = this
+                .WhenAnyValue(x => x.Name)
```

Comment:
Instead of using `WhenAnyValue.Select`, use `WhenAny` and use the Selector at the end

Reply:
I can't seem to make this work.  Can you point to an example?